### PR TITLE
fix: loadSurroundingMessages method incorrect sorting query

### PR DESCRIPTION
**Pull request by aleksandernsilva**
## Proposed changes (including videos or screenshots)
This pull request improves the reliability and correctness of the `loadSurroundingMessages` method by updating its sorting logic and adding comprehensive end-to-end tests. 

Previously the surrounding logic sorted messages before the target in descending order, while the messages after the target were sorted in ascending order. (e.g `msg2 -> msg1 -> msg3 (target) -> msg4 -> msg5`)

The changes ensure that messages are returned in proper chronological order and that the method handles various edge cases and error conditions.

**Testing improvements:**

* Added a new test suite for `loadSurroundingMessages` in `apps/meteor/tests/end-to-end/api/methods.ts`, covering scenarios such as authentication, invalid parameters, non-existent messages, limit handling, and message ordering.
* Updated imports in the test file to include the `sendMessage` helper for more flexible message creation.

**Core logic fixes:**

* Changed the sorting order for fetching surrounding messages in `apps/meteor/server/methods/loadSurroundingMessages.ts` to ensure messages are returned in chronological order (`ts: 1`).
* Removed redundant sorting logic for the "after" messages, relying on the initial options setup for consistency.

## Issue(s)
[CORE-1893](https://rocketchat.atlassian.net/browse/CORE-1893)

## Further comments
#39092

[CORE-1893]: https://rocketchat.atlassian.net/browse/CORE-1893?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Summary by CodeRabbit

* **Bug Fixes**
  * Fixed the ordering of messages returned when loading surrounding messages. Messages now display in correct chronological order for improved conversation readability.

### CODE REVIEW

**Comment by aleksandernsilva** [comment]
@coderabbitai review

File: apps/meteor/server/methods/loadSurroundingMessages.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ Meteor.methods<ServerMethods>({
 
 		const options: FindOptions<IMessage> = {
 			sort: {
-				ts: -1,
+				ts: 1,
```
Comment (KevLehman) [inline apps/meteor/server/methods/loadSurroundingMessages.ts:58]: Image
